# Array of tracked targets
std_msgs/Header header
Target[] targets
